Sources/TuistKit/Commands/Project/ProjectUpdateCommand.swift

```diff
@@ -21,6 +21,11 @@ struct ProjectUpdateCommand: AsyncParsableCommand {
     )
     var defaultBranch: String?
 
+    @Option(
+        help: "Set the connected git repository. Example: --repository https://github.com/tuist/tuist"
+    )
+    var repository: String?
```

Comment:
If we end up leaning on the API taking a URL I'd recommend renaming this to:
```suggestion
    var repositoryURL: String?
```